package.json

```diff
@@ -202,4 +202,4 @@
     "webpack-manifest-plugin": "2.2.0",
     "webpack-merge": "4.2.1"
   }
-}
\ No newline at end of file
+}
```

Comment:
Please remove this change before merging.  Your IDE is fighting with our release script and it's going to really muddy up the git history.